js/views/settingsVw.js

```diff
@@ -113,7 +114,19 @@ module.exports = Backbone.View.extend({
       self.delegateEvents(); //delegate again for re-render
       self.setFormValues();
       self.setState(self.options.state);
-      //self.renderBlocked(self.model.get("user").blocked);
+      
+      // Since the Blocked Users View kicks off many server calls (one
+      // for each blocked user) and since we are re-rendering the entire
+      // settings view often (after each save), we will cache the Blocked
+      // Users View. If for some reason, you really need to re-render it,
+      // call renderBlocked() explicitly after calling render().
```

Comment:
We could do the calls 25 at a time, and if you scroll past the first 25, load 25 more.